### PR TITLE
Code Firefox 100 masthead for mozorg homepage

### PR DESCRIPTION
**Pull request by nathan-barrett**
## One-line summary

Adding the Firefox 100 masthead for the v.100 release

## Significant changes and points to review

Will remove the mr2 promo and add the fx100  promo in its place

## Issue / Bugzilla link
#11539 
## Testing

Demo server URL: (or None)

To test this work:

- [ ] check that the promo is visible when viewing homepage

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/home/home-contentful.html [hunk 1]
```diff
@@ -70,10 +72,15 @@
     <h1>{{ self.page_title() }}</h1>
   </header>
 
-  {% if switch('homepage-mr2-promo') %}
-    {% include 'mozorg/home/includes/mr2-promo.html' %}
+
+  {% if switch('homepage-fx100-promo') %} #}
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/home/home-contentful.html:76]: Cleanup: the leftover end comment is getting rendered on the page.
```suggestion
  {% if switch('homepage-fx100-promo') %}
```

File: bedrock/mozorg/templates/mozorg/home/includes/fx100-promo.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+<section class="fx100-hero">
+  <div class="mzp-l-content mzp-t-content-md">
+  <div class="fx100-hero-title-container">
+    <div class="fx100-hero-wordmark">Firefox 100</div>
+    <h1 class="fx100-hero-title">Building a better internet since 2004</h1>
+  </div>
+  <div class="fx100-hero-cta-container">
+    <a class="mzp-c-button mzp-t-product" href="{{ url('firefox.browsers.index') }}">Get the new Firefox</a>
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/home/includes/fx100-promo.html:13]: Just confirming this should link to the main /browsers page and not /new ?

File: bedrock/mozorg/views.py [hunk 3]
```diff
@@ -125,7 +125,7 @@ def home_view(request):
         "donate_locale": donate_locale,
         "donate_params": donate_params,
         "pocket_articles": PocketArticle.objects.all()[:4],
-        "ftl_files": ["mozorg/home", "mozorg/home-mr2-promo"],
+        "ftl_files": ["mozorg/home", "mozorg/home-mr2-promo", "mozorg/home-fx100-promo"],
```
Comment (craigcook) [inline bedrock/mozorg/views.py:128]: No longer needed if there is no FTL file.
```suggestion
        "ftl_files": ["mozorg/home", "mozorg/home-mr2-promo"],
```

**Review by craigcook — CHANGES_REQUESTED** [review]
Looks good, just needs some minor cleanup. Nice work!

File: bedrock/mozorg/templates/mozorg/home/includes/fx100-promo.html (hunk 2 above)
Comment (nathan-barrett) [reply bedrock/mozorg/templates/mozorg/home/includes/fx100-promo.html:13]: Yeah this is the URL that dan suggested

**Review by craigcook — APPROVED** [review]

**Pull request merged**